docs/docs/configuration/auth.md

```diff
@@ -72,22 +72,42 @@ Note: The user is checked against the group members list on initial authenticati
 
 ### Azure Auth Provider
 
-1. Add an application: go to [https://portal.azure.com](https://portal.azure.com), choose **"Azure Active Directory"** in the left menu, select **"App registrations"** and then click on **"New app registration"**.
-2. Pick a name and choose **"Webapp / API"** as application type. Use `https://internal.yourcompany.com` as Sign-on URL. Click **"Create"**.
-3. On the **"Settings"** / **"Properties"** page of the app, pick a logo and select **"Multi-tenanted"** if you want to allow users from multiple organizations to access your app. Note down the application ID. Click **"Save"**.
-4. On the **"Settings"** / **"Required Permissions"** page of the app, click on **"Windows Azure Active Directory"** and then on **"Access the directory as the signed in user"**. Hit **"Save"** and then then on **"Grant permissions"** (you might need another admin to do this).
-5. On the **"Settings"** / **"Reply URLs"** page of the app, add `https://internal.yourcompanycom/oauth2/callback` for each host that you want to protect by the oauth2 proxy. Click **"Save"**.
-6. On the **"Settings"** / **"Keys"** page of the app, add a new key and note down the value after hitting **"Save"**.
-7. Configure the proxy with
+1. Add an application: go to [https://portal.azure.com](https://portal.azure.com), choose **Azure Active Directory**, select 
+**App registrations** and then click on **New registration**.
+2. Pick a name, check the supported account type(singletenant, multitenant, etc). In the **Redirect URI** section create a new 
+**Web** platform entry for each app that you want to protect by the oauth2 proxy(e.g. 
+https://internal.yourcompanycom/oauth2/callback). Click **Register**.
+3. Next we need to add group read permissions for the app registration, on the **API Permissions** page of the app, click on 
```

Comment:
Additionally on testing this, I was issued a v1 token even on the v2 endpoints.

Do users have to follow these instructions to force v2 tokens to be issued?

https://stackoverflow.com/questions/59790209/issuer-in-access-token-from-azure-active-directory-is-https-sts-windows-net-wh/59799249

Reply:
Yea you do need to set the version in the manifest. Should be called out here.

Reply:
Yup, good catch, adjusted the docs accordingly.

Reply:
Hmm, circling back to this, we still need to add the group read permissions even when using the `/me` endpoint, otherwise we'll get:
```
[2022/07/15 15:26:07] [oauthproxy.go:775] Error creating session during OAuth2 callback: unable to get groups from Microsoft Graph: unexpected status "403": {"error":{"code":"Authorization_RequestDenied","message":"Insufficient privileges to complete the operation.","innerError":{"date":"2022-07-15T12:26:07","request-id":"88ad6c33-3a88-4468-ba89-04a09b42de56","client-request-id":"88ad6c33-3a88-4468-ba89-04a09b42de56"}}}
```

Added that step back into the docs.